gnd/src/main/java/com/google/android/gnd/persistence/local/LocalDataStore.java

```diff
@@ -49,13 +48,13 @@
 public interface LocalDataStore {
 
   /** Load projects stored in local database. */
-  Single<List<Project>> getProjects();
+  Single<ImmutableList<Project>> getProjects();
```

Comment:
👏 👏 

Thanks also for the name fixes, the DAO injection, and all the rest!

Reply:
I even found a bug while testing.